### PR TITLE
Test and fix Dockerfiles

### PR DESCRIPTION
**Pull request by MikeMcQuaid**
- Build x86_64 Dockerfile on Travis CI (ARM one is broken)
- Simplify and update README instructions
- Use Debian Wheezy for both Dockerfiles
- Make Dockerfiles more consistent

### CODE REVIEW

**Comment by MikeMcQuaid** [comment]
@sjackman FYI this was the Dockerfile I used to build the x86_64 container. The other Dockerfile would not build correctly. I'd suggest that once the ARM container is fixed we have the Linux CI here just building inside Docker. Would you be able to pick that up?

**Comment by sjackman** [comment]
Yes, once I'm back from vacation, I can look into that.

File: docker/Dockerfile.x86_64 [hunk 1]
```diff
@@ -1,40 +1,33 @@
...
     HOMEBREW_BUILD_BOTTLE=1 \
     HOMEBREW_BUILD_FROM_SOURCE=1 \
-    HOMEBREW_CC=cc \
     HOMEBREW_FORCE_VENDOR_RUBY=1 \
     HOMEBREW_NO_ANALYTICS=1 \
     HOMEBREW_NO_AUTO_UPDATE=1
 
-RUN echo 'insecure' >> /home/linuxbrew/.curlrc
 RUN git clone --depth=1 https://github.com/Linuxbrew/brew /home/linuxbrew/.linuxbrew && \
     brew analytics off && \
```
Comment (zbeekman) [inline docker/Dockerfile.x86_64]: It looks like linuxbrew is installing portable-ruby 2.3.3_2 when running any `brew` command. This install is failing with:

```
portable-ruby/2.3.3_2/README.md
+ safe_cd /home/linuxbrew/.linuxbrew/Library/Homebrew/vendor/portable-ruby
+ cd /home/linuxbrew/.linuxbrew/Library/Homebrew/vendor/portable-ruby
+ quiet_stderr ./2.3.7/bin/ruby --version
+ [[ -z '' ]]
+ command ./2.3.7/bin/ruby --version
+ rm -rf 2.3.7
+ [[ -d 2.3.7.reinstall ]]
+ odie 'Failed to install vendor ruby.'
+ onoe 'Failed to install vendor ruby.'
+ [[ -t 2 ]]
+ echo -n 'Error: '
Error: + [[ 1 -eq 0 ]]
+ echo 'Failed to install vendor ruby.'
Failed to install vendor ruby.
+ exit 1
Error: Failed to install vendor Ruby.
```

Comment (zbeekman) [reply docker/Dockerfile.x86_64]: (but then it's expecting 2.3.7 to be present and dying when it can't execute 2.3.7)

**Pull request merged**